src/__mocks__/getLikesMock.js

```diff
@@ -0,0 +1,16 @@
+const getLikesMock = async (idMeal) => {
+  const websiteID = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/5OvdDl22dcA0TUpGjcko';
+  const fetchLikes = fetch(`${websiteID}/likes/`);
+  const likesCount = await fetchLikes;
+  const likesJson = await likesCount.json();
+  const result = likesJson.filter((obj) => obj.item_id === idMeal);
+  try {
+    const { likes } = result[0];
+    return likes;
+  } catch {
+    const likes = 0;
+    return (`${likes} there is no data at ${idMeal}`);
+    // return likes;
+  }
+};
+module.exports = getLikesMock;
```

Comment:
- [ ] Please remember to use ES6 syntax, ES6 syntax makes your code more readable along with other advantages. 😸 
    - Please take a look to the rest of your project and change this syntax for ES6 syntax 🚀

Reply:
Using ES6 syntax is a requirement, if you need to transpile these files I recommend you to take a look at this documentation [Using ES6 with jest](https://jestjs.io/docs/getting-started#using-babel) 😸